lib/active_merchant/billing/credit_card_methods.rb

```diff
@@ -234,6 +239,12 @@ def valid_checksum?(numbers) #:nodoc:
 
           sum % 10 == 0
         end
+
+        def checksum_exempt?(number)
+          brand = brand?(number)
```

Comment:
Ugh. Return value from a predicate. (I know this isn't your doing, just generally complaining)